src/System-Platforms/UnixPlatform.class.st

```diff
@@ -106,6 +103,21 @@ UnixPlatform >> lineEnding [
 	^ String lf
 ]
 
+{ #category : #'accessing - linux' }
+UnixPlatform >> linuxIdentificationParameters [
+	"see https://www.freedesktop.org/software/systemd/man/os-release.html"
+
+	| props extractor |
+	props := Dictionary new.
+	self isLinux ifFalse: [ ^props ].
+	extractor := [ '/etc/os-release' asFileReference contents linesDo: [ :each | 
+		props at: (each copyUpTo: $=) asSymbol put: (each copyAfter: $=) ]].
+
+	"Do nothing when it does not exist"
+	[ extractor value ] on: FileDoesNotExistException do: [ self error: 'Only available on modern distributions' ].
```

Comment:
Can we replace "modern distributions" with "distributions supporting the linux standard base" ?

Reply:
@noha  Yes - we can ;)

I changed the comment